spec/features/user_adds_new_game_spec.rb

```diff
@@ -16,8 +16,15 @@
     expect(page).to have_content("wowgame")
   end
 
-  scenario "game details can't be blank" do
+  scenario 'visitors cannot add a new game' do
+    visit "/games/new"
```

Comment:
look `rake routes` to write this the rails way instead of the sinatra way. meaning if you wanted to go to the games index page, your test would contain `visit games_path` instead of `visit /games/index`